src/main.rs

```diff
@@ -72,13 +72,13 @@ mod resource_tests {
 }
 
 fn get_command() -> Command {
-    println!("Type 1 to get scotts testnet balance, Type 2 to get some MOBI info\n");
+    println!("Type \"b\" to get Scott's testnet balance, Type \"m\" to get some MOBI info\n");
 
     let mut command = String::new();
     match io::stdin().read_line(&mut command) {
         Ok(n) if n > 1 => match command.trim() {
-            "1" => Command::ScottsBalance,
-            "2" => Command::MobiInfo,
+            "b" => Command::ScottsBalance,
```

Comment:
what if you provided both?

```
match command.trim() {
    "1" | "b" => 
```
?

Reply:
example:
https://play.rust-lang.org/?gist=db8c54ca6ee93896fafee6716c02c114&version=stable

Reply:
sweet. i'll do